devbox.json

```diff
@@ -16,7 +16,6 @@
     "golines@latest",
     "goreleaser@latest",
     "gotestsum@latest",
-    "govulncheck@latest",
```

Comment:
Why this needs to be moved?

Reply:
Needs to be built with go 1.23 to pick up go 1.23 vulnerabilities... nix by default builds with go 1.21 iirc so have to do custom override.

Reply:
Thanks